src/UpdatesPublisher.js

```diff
@@ -62,11 +83,29 @@ class UpdatesPublisher {
       await this.start()
     })
 
-    this._changes.on('data', async (data) => {
-      logger.info('changes detected')
-      await this._producer.sendMessage(data)
-      logger.info('data: ' + JSON.stringify(data))
-      logger.info('successfully published changes')
+    changesStream.on('data', async ({ doc: npmDoc }) => {
+      const {
+        name,
+        'dist-tags': distTags
+      } = npmDoc
+
+      const { latest: version } = distTags
+
+      logger.info(`Received update for package: ${name}, version: ${version}`)
+
+      const update = DependencyUpdate.wrap({
```

Comment:
I think we should still wrap and check for errors here. We shouldn't bother publishing if errors occurred and should definitely log them.